### ISSUE TITLE
vim like keybindings not working on mac

### ISSUE THREAD

**Issue by jacobono** [extrawurst/gitui#1491]
following https://github.com/extrawurst/gitui/blob/master/KEY_CONFIG.md

i would expect hjkl keys to work.  Instead 'h' always brings up help menu and no motion with the other keys.

running with '-l' flag and i don't see a log about reading the key config file.  makes sense as I threw in an 

```
edit_file: Some(( code: Char('I'), modifiers: ( bits: 1,),)),
```

and  'e' is what triggers the edit action, not (shift i) 'I'.

these are the files in the config dir
                                                                                                                                                                          
/Users/jacobono/.config/gitui

- key_bindings.ron
- key_config.ron
- theme.ron

am i doing something silly (?) bc the instructions are fairly straightforward and i've double checked a couple of times.

mac: 12.6 monterrey
alacritty 0.10.1 (2844606)
gitui 0.22.1

**Comment by extrawurst** [comment]
are you sure you rebound the help key away from `h`? see https://github.com/extrawurst/gitui/blob/master/vim_style_key_config.ron#L21

**Comment by jacobono** [comment]
ahhh yes so i misunderstood the 

'// every key not overwritten via the config file will use the default specified there'

i read that as key like key stroke, not key in the struct.  thank you, now it works!

when the motion didn't work i realized that i syntax error for comment.  then it loaded, but no way to tell that there was an error parsing the .ron file when logging.

is that something you might want as a PR?

**Comment by extrawurst** [comment]
hm yeah some error in the log would make sense indeed

**Comment by jacobono** [comment]
ok, ill look at making a PR.  i was thinking something like 

**log** 
- reading config file at _path_
if found, **log** read config file successfully
if not found **log** no config file, using defaults (maybe logging defaults?)
if error in config, spit out error, **log** defaults.  looking at KeyListFile::read_file.

cool?

**Comment by extrawurst** [comment]
lets have a PR to start talking about 👍

**Comment by extrawurst** [comment]
closed via 32f6134
